lib/src/model/robotoff_nutrient_extraction.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'per_size.dart';
+import 'robotoff_nutrient_extraction_annotation.dart';
+import '../interface/json_object.dart';
+import 'nutrient.dart';
+import 'robotoff_nutrient_extraction_insight.dart';
+
+part 'robotoff_nutrient_extraction.g.dart';
+
+@JsonSerializable()
+class RobotoffNutrientExtractionResult extends JsonObject {
+  final String? status;
+  final int? count;
+  final List<RobotoffNutrientExtractionInsight>? insights;
+  RobotoffNutrientExtractionInsight? _latestInsight;
+
+  RobotoffNutrientExtractionResult({
+    this.status,
+    this.count,
+    this.insights,
+  });
+
+  RobotoffNutrientExtractionInsight? getLatestInsights({
+    bool recompute = false,
```

Comment:
I would remove that "recompute" possibility, because I can't see a good reason to recompute.